feat: add existingSecret value to override secret with ext secrets

josephvano opened the pull request:
* add helper methods to create secret
* add helper method to get secre name
* update refs for secret to use helper method

Signed-off-by: Joseph Vano <joseph.vano@gmail.com>

<!--

* https://github.com/helm/helm/tree/master/docs/chart_best_practices

Following best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

#### What this PR does / why we need it:

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

Jasstkn reviewed (changes requested):
@josephvano hi! thank you for your contribution!

To start review process, please, update your branch from the master and resolve any conflicts on the way.

Jasstkn commented:
> Hi @Jasstkn I've rebased from master. Thanks

I see you have a failed job about updating docs. Did you check our contributing guide?
https://github.com/litmuschaos/litmus-helm/blob/master/CONTRIBUTING.md#how-to-contribute

josephvano commented:
Hi @Jasstkn I've updated the helm-docs and rebased again.

I'm not sure why there is an error with the `lint-charts` check with validation of maintainers.

Jasstkn commented:
> Hi @Jasstkn I've updated the helm-docs and rebased again.
> 
> I'm not sure why there is an error with the `lint-charts` check with validation of maintainers.

Thanks. I'm fixing the issue right now. :)

Jasstkn reviewed (approved):
Hi @josephvano! Again thank you for your contribution. LGTM :)

Jasstkn commented:
@ispeakc0de @imrajdas Hi folks. Please, review this PR. :)

Jasstkn commented:
> @ispeakc0de @imrajdas Hi folks. Please, review this PR. :)

@ispeakc0de @imrajdas ping

agcorrea commented:
+1

josephvano commented:
Rebased

ready for review again @ispeakc0de @imrajdas

Jonsy13 commented on charts/litmus/Chart.yaml:
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart to install ChaosCenter
 name: litmus
-version: 2.10.2
+version: 2.11.0
```
Can we please change it to **`2.10.3`**

Jonsy13 commented on charts/litmus/README.md:
```diff
@@ -1,6 +1,6 @@
 # litmus
 
-![Version: 2.10.2](https://img.shields.io/badge/Version-2.10.2-informational?style=flat-square) ![AppVersion: 2.10.0](https://img.shields.io/badge/AppVersion-2.10.0-informational?style=flat-square)
+![Version: 2.11.0](https://img.shields.io/badge/Version-2.11.0-informational?style=flat-square) ![AppVersion: 2.10.0](https://img.shields.io/badge/AppVersion-2.10.0-informational?style=flat-square)
```
Can we please change it to **`2.10.3`**
